src/telegram.js

```diff
@@ -1467,6 +1467,19 @@ class TelegramBot extends EventEmitter {
     form.ok = ok;
     return this._request('answerPreCheckoutQuery', { form });
   }
+
+  /**
+   * Use this method to get the sticker set given the name of the set.
+   * Returns a [StickerSet](https://core.telegram.org/bots/api#stickerset) object.
+   *
+   * @param {String} name Name of the sticker set
```

Comment:
2 spaces after `@param` please. It makes the type clauses e.g. `{String}` align perfectly with that of `@return`.